Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -51,3 +51,17 @@ header: <wtf/text/AtomString.h>
     uint64_t high();
     [Validator='((static_cast<UInt128>(*high) << 64) | *low) != WTF::UUID::deletedValue'] uint64_t low();
 }
+
+#if OS(WINDOWS)
+using WTF::ProcessID = int;
+#else
```

Comment:
I don't think generate-serializers.py supports this.  I think you'll need #endif #if !OS(WINDOWS)

Reply:
generate-serializers.py updated in https://github.com/WebKit/WebKit/pull/17940